meteoritics-and-planetary-science.csl

```diff
@@ -125,8 +129,8 @@
   </citation>
   <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="1">
     <sort>
-      <key macro="author"/>
-      <key variable="title"/>
+      <key macro="author-short" names-use-first="1" names-min="2"/>
+      <key macro="year-date"/>
```

Comment:
@adam3smith, it shouldn't really matter now for the style output, but shouldn't the bibliography sort on the "author" macro instead of the "author-short" one?